xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -14956,11 +14956,11 @@ glusterd_add_peers_to_auth_list(char *volname)
                "auth allow list is not set");
         goto out;
     }
+    len = strlen(auth_allow_list);
     cds_list_for_each_entry_rcu(peerinfo, &conf->peers, uuid_list)
     {
-        len += strlen(peerinfo->hostname);
+        len += strlen(peerinfo->hostname) + 1;
```

Comment:
A comment mentioning the reason to do so will be helpful for people reading the code in the future.

Reply:
I have amend and re-pushed the commit, hopefully addressing the concerns

Reply:
The commit-msg for the PR also needs to be updated